tests/bench/qsort/.github/workflows/lean_action_ci.yml

```diff
@@ -0,0 +1,14 @@
+name: Lean Action CI
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: leanprover/lean-action@v1
```

Comment:
Left over file?

Reply:
Actually, `lake new` creates this now, and I just checked everything in. I'll remove it.